[2025-03-09T17:38:50.598+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweather_data_lake.data_to_raw manual__2025-03-09T17:38:47.927602+00:00 [queued]>
[2025-03-09T17:38:50.602+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweather_data_lake.data_to_raw manual__2025-03-09T17:38:47.927602+00:00 [queued]>
[2025-03-09T17:38:50.602+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 3
[2025-03-09T17:38:50.608+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): data_to_raw> on 2025-03-09 17:38:47.927602+00:00
[2025-03-09T17:38:50.612+0000] {standard_task_runner.py:57} INFO - Started process 3641 to run task
[2025-03-09T17:38:50.615+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'openweather_data_lake', 'data_to_raw', 'manual__2025-03-09T17:38:47.927602+00:00', '--job-id', '444', '--raw', '--subdir', 'DAGS_FOLDER/openweather_data_lake.py', '--cfg-path', '/tmp/tmpqfnh1ixg']
[2025-03-09T17:38:50.618+0000] {standard_task_runner.py:85} INFO - Job 444: Subtask data_to_raw
[2025-03-09T17:38:50.630+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-09T17:38:50.657+0000] {task_command.py:415} INFO - Running <TaskInstance: openweather_data_lake.data_to_raw manual__2025-03-09T17:38:47.927602+00:00 [running]> on host 405dcb727823
[2025-03-09T17:38:50.711+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweather_data_lake' AIRFLOW_CTX_TASK_ID='data_to_raw' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T17:38:47.927602+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T17:38:47.927602+00:00'
[2025-03-09T17:38:52.357+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-09T17:38:52.383+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=openweather_data_lake, task_id=data_to_raw, execution_date=20250309T173847, start_date=20250309T173850, end_date=20250309T173852
[2025-03-09T17:38:52.415+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-09T17:38:52.448+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
